src/components/History/History.tsx

```diff
@@ -62,13 +63,21 @@ const History: React.FC<HistoryProps> = ({
     return sortArrayByDate(history)
   }, [history])
 
+  useEffect(() => {
+    trackEvent('HL Chat Conversation Selected', { conversationId: history.length });
+  }, [history.length]);
```

Comment:
This seems unintended? history.length as conversationId? This event would fire every time history length sends, which isn't quite what the event would indicate?

Reply:
Actually this is useless removed this